recipes/mariadb/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "mariadb" %}
+{% set version = "1.1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mariadb-{{ version }}.zip
+  sha256: 73a0acbd2ace381ec1bcfc61ced7a79a519e66c02c24e12ae6d27baa03717876
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: True
+  include_recipe: True
+
+requirements:
+  host:
```

In order to compile:
```suggestion
  build:
    - {{ compiler('c') }}
  host:
```